### ISSUE TITLE
Official Docker Hub image

### ISSUE THREAD

**Issue by unacceptable** [aquasecurity/cloudsploit#437]
It would be nice if I could incorporate cloudsploit into my CD pipeline in a similar fashion to how I incorporated Trivy into my CI pipeline.  Looking at this repo, this is already containerized, we just need an official image pushed to docker hub.

I definitely get that I could build and maintain my own image in my own docker repository (and I will do that until there's an official one), but that tends to lead towards Tech Debt in my stack.

**Comment by exigopro** [comment]
Indeed, it would also alleviate some manual work on our side of keeping the container up to date with the latest stable release each time.
